index.js

```diff
@@ -11,7 +12,15 @@ const resolveFrom = (fromDir, moduleId, silent) => {
 		throw new TypeError(`Expected \`moduleId\` to be of type \`string\`, got \`${typeof moduleId}\``);
 	}
 
-	fromDir = path.resolve(fromDir);
+	try {
+		fromDir = fs.realpathSync(fromDir);
+	} catch (err) {
+		if (err.code !== 'EINVAL') {
```

Comment:
@sindresorhus, after thinking about this more, I am not so sure we have to throw any error. Right now it works if the `fromDir` does not exist in the filesystem. [resolve](https://github.com/substack/node-resolve) seems to behave the same. So maybe we just need to use `path.resolve` on `fromDir` and then ignore `ENOENT` errors of `realpathSync`. What do you think?